### ISSUE TITLE
Error when using esm.sh

### ISSUE THREAD

**Issue by christianmagill** [barvian/fluid-tailwind#13]
I've recently over the past couple days started receiving an error when using Winden/Yabe Suil for Wordress

`import { fluidExtractor, fluidCorePlugins, defaultThemeScreensInRems, defaultThemeFontSizeInRems } from 'https://esm.sh/fluid-tailwind?bundle-deps&external=fs';`

`Config Error
SyntaxError: The requested module '/v135/tailwindcss@3.4.1/X-ZS9mcw/es2022/lib/corePlugins.js' does not provide an export named 'corePlugins'`

It was previously working.

**Comment by barvian** [comment]
Hmm, it looks like a regression in the last update. Could you try `https://esm.sh/fluid-tailwind@0.1.6?bundle-deps&external=fs` for now? I'll look into the cause and try to push a fix in the meantime 🤔

**Comment by christianmagill** [comment]
Yes, reverting to the previous version fixes the issue.

**Comment by krstivoja** [comment]
+1

**Comment by barvian** [comment]
I've opened [an issue](https://github.com/esm-dev/esm.sh/issues/826) with ESM.sh because I can't seem to find what changed between versions to cause this breakage 🤔

**Comment by barvian** [comment]
This should be fixed with v0.3.6. Here's [a working StackBlitz](https://stackblitz.com/edit/stackblitz-starters-wxvznu?file=index.html) that should be adaptable to Winden/Yabe Suil. Note the (currently undocumented) option passed to the plugin: 

```
plugins: [
  fluid({
    checkExtractConfig: false,
  }),
],
```

This was necessary because it seems Tailwind Play ignores or overrides the extractor passed in (though it works without one).

Feel free to comment if you're still experiencing issues!

**Comment by krstivoja** [comment]
Awesome stuff @barvian. Updated our documentation for Winden and I can confirm that this setup is working now: https://docs.dplugins.com/winden/fluid-tailwind/
